pkg/tfvars/ovirt/ovirt.go

```diff
@@ -73,6 +79,21 @@ func TFVars(
 		}
 		cfg.BaseImageLocalFilePath = imageFilePath
 	}
-
+	if len(affinityGroups) > 0 {
+		cfg.AffinityGroups = handleAffinityGroups(affinityGroups, infraID)
+	}
 	return json.MarshalIndent(cfg, "", "  ")
 }
+
+func handleAffinityGroups(ags []*ovirt.AffinityGroup, infraID string) []map[string]interface{} {
+	var agMap []map[string]interface{}
+	for _, ag := range ags {
+		agMap = append(agMap, map[string]interface{}{
+			"name":        fmt.Sprintf("%s-%s", infraID, ag.Name),
+			"priority":    float64(ag.Priority),
```

Comment:
Why does this need to be converted to a float? Is it necessary for the way that the value gets marshaled?

Reply:
Yes the oVirt API expects a float so the terraform provider expects a float

Reply:
Terraform does not distinguish between whole numbers and fractional values.